### PR TITLE
[grid-template-areas] Supported grid-template(-areas) in media rules

### PR DESCRIPTION
**Pull request by evgeny-petukhov**
Problem is mentioned in the issues #954 #1016 and discussed in the issue #954

### CODE REVIEW

File: test/cases/grid-template-areas.out.css [hunk 1]
```diff
@@ -32,32 +32,20 @@
...
     .a {
         grid-template-areas: "head main";
     }
-
     .b {
-        -ms-grid-row: 1;
-        -ms-grid-column: 1;
-        grid-area: head;
+        -ms-grid-row:
+        1;
```
Comment (evgeny-petukhov) [inline test/cases/grid-template-areas.out.css]: There is a little problem here with whitespaces. Is it needed to merge raws from different rules/decls for the better result?

Comment (ai) [reply test/cases/grid-template-areas.out.css]: Just trim spaces around value and remove `raws.between`

Comment (evgeny-petukhov) [reply test/cases/grid-template-areas.out.css]: Used `raws.between` of the origin declaration

File: test/cases/grid-template.out.css [hunk 2]
```diff
@@ -91,3 +91,34 @@
     -ms-grid-column-span: 9;
     grid-area: ft;
 }
+
+@media (min-width: 1000px) {
+    .b {
+        -ms-grid-rows:
+            30px auto 1fr 30px;
```
Comment (ai) [inline test/cases/grid-template.out.css]: I think we should not have new line here and l in next property. Let's clean `between` here too

Comment (evgeny-petukhov) [reply test/cases/grid-template.out.css]: It's a little different case. These declarations extends `between` from origin which already had new line. So it's better to control further fixes with user's styleguide or linter. But I can fix test and remove new line from origin.

Comment (evgeny-petukhov) [reply test/cases/grid-template.out.css]: @ai What do you think?

Comment (evgeny-petukhov) [reply test/cases/grid-template.out.css]: @ai About my second commet

Comment (ai) [reply test/cases/grid-template.out.css]: Just remove `between`. Do not copy it from an origin. PostCSS will fill it automatically with own space detect algorithm.

**Comment by ai** [comment]
Released in 8.3.

**Pull request merged**